src/views/Dashboard/GeneticAnalyst/Home/index.vue

```diff
@@ -204,12 +211,16 @@ export default {
             })
           }
 
-          orders.push(data)
+          if (page.name == "ga-orders" && data.status == "Done") orders.push(data)
+          else if (page.name == "ga-dashboard" && data.status != "Done") orders.push(data)
```

Comment:
I'm just curious, can you explain how that logic works?

Reply:
this page used in `ga-orders` & `ga-dashboard` but in different filter data. Orders page only show `status = Done` (first condition), while on the dashboard page showing other than `Done` (second condition)

Reply:
Hmm, I don't think adding global logic like this is good, can you please refactor this?

Reply:
updated to using global component and send params for the filter @agustinustheo , wdyt?

Reply:
i also update updated the unit testing because there was an error